storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -31,5 +31,15 @@ include::modules/persistent-storage-csi-about.adoc[leveloffset=+1]
 
 include::modules/persistent-storage-csi-vsphere-stor-policy.adoc[leveloffset=+1]
 
+[NOTE]
+====
+If you want to update your cluster to {product-title} 4.11 or later and have previously installed an upstream vSphere CSI Operator Driver, you must remove the upstream driver before updating the cluster.
+====
```

Comment:
same as above. I am not comfortable with dropping wording for 4.11 without adding additional context.